packages/system-next/src/Collections/Immutable/SortedMap/index.ts

```diff
@@ -0,0 +1,173 @@
+import { pipe } from "@effect-ts/system/Function"
```

Comment:
This should be a relative import `"../../../Function"`.

Reply:
nice catch!